tests/ZendTest/Code/Generator/TestAsset/TestSampleSingleClassMultiLines.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * File header here
+ *
+ * @author Ralph Schindler <ralph.schindler@zend.com>
+ */
+
+namespace ZendTest\Code\Generator\TestAsset;
+
+/**
+ * class docblock
```

Comment:
imo, you can change "class docblock" text with something like "This class test single class with multi lines"...